app/screens/home/StoreList.java

```diff
@@ -1,26 +1,68 @@
 package icebase.app.screens.home;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 import icebase.app.App;
+import icebase.app.api.API;
+import icebase.app.enums.SCREEN_ENUM;
 import icebase.app.MenuTitle;
+import icebase.app.Router;
 import icebase.app.screens.Screen;
+import icebase.app.types.Store;
+import icebase.icebase.Auth;
+import icebase.icebase.User;
 
 public class StoreList implements Screen {
     public void display() {
         Scanner sc = App.sc;
-
-        MenuTitle.displayMainTitle();
-        MenuTitle.displaySubTitle("[STORES]");
-        String[] storeList = { "Store 1", "Store 2" };
-        int range = storeList.length;
-        storeList[range - 1] = "Return";
         int choice;
+        Store chosenStore;
+        Auth currentUser = Auth.getAuth();
```

Comment:
doesnt match sa variable name `currentUser`

i think better

```
Auth auth = Auth.getAuth();
User currentUser = auth.getUser();
String currentUserId = currentUser.getId()
```

Reply:
oki thank youu! but like pwede ba diretso na lang na ganto

`String currentUserID = Auth.getAuth().getUser().getId();`

Reply:
i think it's better na to get the instances of auth and current user since maybe ma use siya further in the code also,,, mas readable for me ang separate vars as compared to method chaining

Reply:
> oki thank youu! but like pwede ba diretso na lang na ganto
> 
> `String currentUserID = Auth.getAuth().getUser().getId();`

Yep same thoughts kay andry, better nga i separate ang different instances rather than calling it by chaining

Reply:
oki thank you